decentraland/unity-renderer
# style: scroll bar width size

Currently, the scroll bar texture is too thin and very hard to see.
Now there are two different textures: one called HandleThick and the other HandleThin. Depending the use case we must use one or another: thick is for full screen views; thin for small UI such as social windows or dropdown menus.

Conversations view
<img width="521" alt="249542861-e135c340-54ca-4dd6-bd7d-a450f518fda4" src="https://github.com/decentraland/unity-renderer/assets/51088292/0d8ba05e-4e54-470f-86b3-28e46b565a47">

Main Menu
<img width="258" alt="249542873-8ca8a04c-6a92-4dc1-b5aa-2f5180c8ee84" src="https://github.com/decentraland/unity-renderer/assets/51088292/0f3f9ba8-70c4-4b73-ac45-127e8138eebf">


How to test?
1- Open https://play.decentraland.org/?explorer-branch=style/ScrollBarHandle
2- Go to the main menu and check places/events/favourites sections now have a thicker handle in the scroll bar.
3- Then open all the windows from the social bar. Confirm the same than above, a thicker handle.
